test/e2e/dra/dra.go

```diff
@@ -1368,13 +1427,17 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 		v1beta2Tests()
 	})
 
+	ginkgo.Context("with partitionable devices", func() {
+		partitionableDevicesTests()
+	})
```

Comment:
```suggestion
	ginkgo.Context("with partitionable devices", partitionableDevicesTests)
```

Reply:
I have updated this and the other places with a similar pattern.